src/unicode-properties.js

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
I’d be interested in hearing why you decided against consuming https://github.com/mathiasbynens/unicode-match-property-value-ecmascript.

The current approach seems like a maintenance problem, which could be alleviated by having `src/unicode-properties.js` be auto-generated by a script. But that is pretty much what https://github.com/mathiasbynens/unicode-match-property-value-ecmascript is doing, so `GOTO 0`.

Reply:
I am okay with adding @mathiasbynens's library as a dependency.